docs/modules/appendix/pages/glossary.adoc

```diff
@@ -109,6 +109,11 @@ ILL (Inter-Library Loan)::
   Inter-Library Loan is the process of one libray borrows materials for a patron from another library. 
 [[IIA]]Import item attributes::
   Used to map the data in your holdings tag to fields in the item record during a MARC import.
+[[index]]Index::
+  verb. to create an index.  
```

Comment:
You can add a `+` to the end of this line so that there is a line break before moving on to the noun definition.